lerna.json

```diff
@@ -1,6 +1,6 @@
 {
     "version": "independent",
-    "packages": ["packages-common/*", "packages-native/*", "packages-web/*"],
+    "packages": ["packages-common/*", "packages-native/*", "packages-web/*", "packages-hybrid/*"],
```

Comment:
This should be common.. As common in this case is native+web otherwise nanoflow-commons should be moved to another package

Reply:
Common.... are capable on the web, Native and Hybrid. Though hybrid is Hybrid only